### PR TITLE
Add a getting started template for resource picking

### PR DESCRIPTION
**Pull request by acearun**

### CODE REVIEW

File: Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Picking a set of resources to analyze in workbooks
+
+The `Resource Picker` template gets you started with subscription, resource group and resource parameters to set up the input context of your workbook. The default parameters are set to pick virtual machines, but you can configure it to pick any type of resource. The template also has a ARG query control that shows you how to use the parameters in your analyses.
+
+![Image](Full.png)
```
Comment (vutran01) [inline Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md]: should this be a full path to the image in github?

File: Workbooks/Azure Monitor - Getting Started/Resource Picker/Resource Picker.workbook [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+          "filter": true
+        }
+      },
+      "showPin": true,
+      "name": "Resource information"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "💡 [Learn how to use this template](https://github.com/microsoft/Application-Insights-Workbooks/tree/master/Workbooks/Azure%20Monitor%20-%20Getting%20Started/Resource%20Picker)"
```
Comment (vutran01) [inline Workbooks/Azure Monitor - Getting Started/Resource Picker/Resource Picker.workbook]: should the readme.md file be here?

File: Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Setting up the resource type to pick
+
+1. Hit `Edit` on the workbook toolbar.
+2. You will now be able to see a drop down `Resource type` before `Subscriptions`:
+
+    ![Image](Parameter.png)
```
Comment (vutran01) [inline Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md]: fullpath?

File: Workbooks/Azure Monitor - Getting Started/Resource Picker/Resource Picker.workbook (hunk 2 above)
Comment (acearun) [reply Workbooks/Azure Monitor - Getting Started/Resource Picker/Resource Picker.workbook]: I think this works, will confirm and change in a separate change if needed.

File: Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md (hunk 3 above)
Comment (acearun) [reply Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md]: Not needed

File: Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md (hunk 1 above)
Comment (acearun) [reply Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md]: Relative path should work.

**Pull request merged**